docs/installing/adding_devices.md

```diff
@@ -24,19 +24,27 @@ The following instructions guide you through the process of installing the requi
 
 {{site.data.keyword.ieam}} supports architectures and operating systems with the following hardware architectures:
 
-* {{site.data.keyword.linux_bit_notm}} devices or virtual machines that run Ubuntu 20.x (focal), Ubuntu 18.x (bionic), Debian 10 (buster), Debian 9 (stretch)
-* Red Hat Enterprise Linux&reg; 8.2
-* Fedora Workstation 32
-* CentOS 8.2
-* SuSE 15 SP2
-* {{site.data.keyword.linux_notm}} on ARM (32-bit), for example Raspberry Pi, running Raspbian buster or stretch
-* {{site.data.keyword.linux_notm}} on ARM (64-bit), for example NVIDIA Jetson Nano, TX1, or TX2, running Ubuntu 18.x (bionic)
-* {{site.data.keyword.macOS_notm}}
+* x86_64
+   * {{site.data.keyword.linux_bit_notm}} devices or virtual machines that run Ubuntu 20.x (focal), Ubuntu 18.x (bionic), Debian 10 (buster), Debian 9 (stretch)
+   * {{site.data.keyword.rhel}} 8.1, 8.2, and 8.3
+   * Fedora Workstation 32
+   * CentOS 8.1, 8.2, and 8.3
+   * SuSE 15 SP2
+* ppc64le
+   * {{site.data.keyword.linux_ppc64le_notm}} devices or virtual machines that run Ubuntu 20.x (focal) or Ubuntu 18.x (bionic)
+   * {{site.data.keyword.rhel}} 7.6, 7.9, 8.1, 8.2, and 8.3
+* ARM (32-bit)
+   * {{site.data.keyword.linux_notm}} on ARM (32-bit), for example Raspberry Pi, running Raspbian buster or stretch
+* ARM (64-bit)
+   * {{site.data.keyword.linux_notm}} on ARM (64-bit), for example NVIDIA Jetson Nano, TX1, or TX2, running Ubuntu 18.x (bionic)
+* Mac
+   * {{site.data.keyword.macOS_notm}}
 
```

Comment:
We likely need to add RISC-V

Reply:
True - Should we have a separate GH Issue to add RISC-V as a supported architecture?

Reply:
yes, please.  And we may want to add a risk-v label, what do you think?